findphotodates.py

```
@@ -190,6 +190,145 @@ def _clear_progress_line():
+    def stop(self):
+        self._stop.set()
+        if self._thread is not None:
+            try:
+                self._thread.join(timeout=1.0)
+            except KeyboardInterrupt:
+                pass
+        if self._restore_posix_terminal is not None:
+            try:
+                self._restore_posix_terminal()
+            except Exception:
+                pass
```

Ruff (0.15.12)

[error] `try`-`except`-`pass` detected, consider logging the exception

(S110)